src/main/java/appeng/core/sync/packets/PacketPartialItem.java

```diff
@@ -20,50 +20,40 @@
 
 
 import io.netty.buffer.ByteBuf;
-import io.netty.buffer.Unpooled;
 
-import net.minecraft.entity.player.EntityPlayer;
+import cpw.mods.fml.common.network.simpleimpl.MessageContext;
 
 import appeng.container.AEBaseContainer;
 import appeng.core.sync.AppEngPacket;
-import appeng.core.sync.network.INetworkInfo;
+import appeng.core.sync.AppEngPacketHandler;
 
 
-public class PacketPartialItem extends AppEngPacket
+public class PacketPartialItem implements AppEngPacket, AppEngPacketHandler<PacketPartialItem, AppEngPacket>
 {
 
-	final short pageNum;
-	final byte[] data;
+	private short pageNum;
+	private byte[] data;
 
 	// automatic.
-	public PacketPartialItem( ByteBuf stream )
+	public PacketPartialItem()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)